test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -251,7 +251,7 @@ export class DisplayTestApp {
     if (ProcessDetector.isElectronAppFrontend) {
       await ElectronApp.startup(opts);
     } else if (ProcessDetector.isIOSAppFrontend) {
-      await IOSApp.startup(opts);
+      await IOSApp.startup(opts as IOSAppOpts);
```

Comment:
Why did you need to cast this?

Reply:
This is because I updated IOSAppOpts to prohibit the passing of an authorizationClient. Even though one isn't provided here, the cast can't be done implicitly.

Reply:
Wasn't that change supposed to get moved to MobileAppOpts, with IOSAppOpts deleted?

Reply:
Yes, though I was planning on doing that in a separate PR. @calebmshafer, should I instead include those changes here?

Reply:
Is it needed for 3.2 release?